packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1713,7 +1713,7 @@ describe('shallow', () => {
 
       it('works without memoizing', () => {
         const wrapper = shallow(<RendersApp />);
-        expect(wrapper.debug()).to.equal('<App />');
+        expect(wrapper.debug()).to.equal(is('>= 17') ? '<AppMemoized />' : '<App />');
```

Comment:
I don't understand this test. Are we just testing `React.memo()` here and the behavior of that method has changed in 17? Why are we testing `React.memo()` in the first place?

It seems like `React.memo()` used to return a copy of the component prior to 17, but now it returns the component by reference.

Reply:
We're testing "what enzyme does with `React.memo`", and thus what `.debug` will print out. The behavior did change in 17, which is why this change is here.

Reply:
Ok. I humbly suggest that we drop this test unless we are wrapping or otherwise building upon `React.memo` with enzyme. It doesn't make sense to me that we should be writing unit tests for straight `React` inside `enzyme`.

Reply:
we have tons of tests for react inside enzyme, and it's important to have tests for how enzyme handles every single react feature.

Why does this need to be dropped, versus just adapted for react 17?

Reply:
Architecturally, tests for React belong in the React project itself and tests for enzyme belong in the enzyme project. If we had an enzyme `memo` method, testing it would make sense, but if we're just writing arbitrary tests for React it doesn't make a lot of sense.

Those are just my thoughts. It's just a suggestion.